txpool/account.go

```diff
@@ -151,6 +157,9 @@ type account struct {
 	enqueued, promoted *accountQueue
 	nextNonce          uint64
 	demotions          uint
+
+	//	maximum number of enqueued transactions
+	maxEnqueued uint64
```

Comment:
Why each account has `maxEnqueued`? `maxEnqueued` is settable for each account? You can pass the pointer and read it by atomic.Load. Or give function object to `account`

Reply:
Nice catch. 

I thought about how to circumvent this - the proper way to extend the `accountQueue` struct to have a limited queue as well. That way I could reject a tx based on the the length of the queue.

If I'm not mistaken, you're suggesting a global variable that can be read atomically. In that case, tests could not be run in parallel, as each test would unpredictably mutate this field, causing tests to randomly fail.

I realize this increases the memory limit, but I intend to rework `account` struct for TxPool v3 (so it's just one field).